R/run_trial.R

```diff
@@ -149,6 +150,49 @@ run_trial <- function(
       )
     }
 
+    # post-processing to get evaluation metrics
+    if(design$evaluation$type %in% target_types_auc) {
```

Comment:
```suggestion
    if(isTRUE(design$evaluation$type %in% target_types_auc)) {
```
I think this would be enough to allow for a NULL evaluate design, which would simply return the default return objects, and therefore the user wouldn't have to specify an additional argument?